src/components/PacksReadme/PacksReadme.tsx

```diff
@@ -252,8 +253,29 @@ export default function PacksReadme() {
     }, [packName]);
 
     useEffect(() => {
-      const searchParams = window ? new URLSearchParams(window.location.search) : null;
+      if (isBrowser && fragmentIdentifier && componentsMounted) {
+        const elementId = fragmentIdentifier.replace("#", "");
+        const parent = document.getElementById?.(elementId);
+        parent?.querySelector?.("a")?.click();
+      }
+    }, [fragmentIdentifier, componentsMounted]);
+
+    useEffect(() => {
+      if (packData) {
+        setTimeout(() => {
+          isComponentsMounted(true);
```

Comment:
```useEffect(() => {
  if (isBrowser && fragmentIdentifier) {
    const timeoutId = setTimeout(() => {
      const elementId = fragmentIdentifier.replace("#", "");
      const parent = document.getElementById?.(elementId);
      parent?.querySelector?.("a")?.click();
    }, 1000);

    return () => clearTimeout(timeoutId);
  }
}, [fragmentIdentifier]);```

If this work. We can get rid of `componentsMounted`

Reply:
it works.. done! the state variable is not needed